x-pack/plugins/security/server/authentication/authentication_service.test.ts

```diff
@@ -966,7 +966,6 @@ describe('AuthenticationService', () => {
         const { authenticator, onPreResponseHandler } = getService();
```

Comment:
__note:__ would be great to add unit tests for new cases in `registerOnPreResponse` if we don't have them yet.